### ISSUE TITLE
idea: example for .ics or .vcf handling

### ISSUE THREAD

**Issue by folovco** [koaning/calmcode-feedback#202]
Don't forget to add a link (or a screenshot) of what needs a-fixin.

**Comment by koaning** [comment]
Could you add some context? It's a bit unclear what you're referring to here.

**Comment by folovco** [comment]
Yeah, this currently seems fairily confusing what we wanted to point to us too.

A good guess is some example similar to https://calmcode.io/qr-code/generate.html , where we can create calendar event and contact files in the most widely used formats.
